lib/queries/index.js

```diff
@@ -0,0 +1,5 @@
+const fs = require('fs')
+
+module.exports.REPO_INFO_QUERY = fs
+  .readFileSync(`${__dirname}/repo_info.graphql`)
+  .toString()
```

Comment:
We'll be doing this very often, might as well just create a function to load `.graphql`

Probably put that in `utils.js` file or there might be an existing npm packages to do it.

Reply:
Sounds good. It'll be like 3 lines of code to do it ourselves so may as well keep it light weight.